webviz_subsurface/containers/_inplace_volumes_onebyone.py

```diff
@@ -139,6 +140,55 @@ def add_webvizstore(self):
             ),
         ]
 
+    @property
+    def tour_steps(self):
+        return [
+            {
+                "id": self.graph_id,
+                "content": "The chart shows inplace volumetrics results. ",
+            },
+            {
+                "id": self.table_id,
+                "content": "The table shows statistics per sensitivity. \
+                            Rows can be filtered by searching, and sorted by \
+                            clicking on a column header. ",
+            },
+            {
+                "id": self.response_id,
+                "content": "Select the volumetric calculation to display. ",
+            },
+            {
+                "id": self.plot_type_id,
+                "content": "Controls the type of the visualized chart. \
+                            Per realization shows bars per realization, \
+                            while the boxplot shows the range per sensitivity. ",
+            },
+            {
+                "id": self.tornadowrapper_id,
+                "content": "Tornado plot for the currently displayed data. \
+                            Differences references can be set and sensitivities \
+                            smaller than the reference can be filtered out. \
+                            Click on the bar of a sensitivity to highlight the \
+                            relevant realizations in the main chart.",
+            },
+            {
+                "id": self.ensemble_id,
+                "content": "If several ensembles are available, the active ensemble \
+                            can be selected here. ",
+            },
+            {
+                "id": self.source_id,
+                "content": "If volumes have been calculated for different grids. \
+                            E.g. geogrid and eclipsegrid, the active grid can be selected here. ",
+            },
+            {
+                "id": self.filter_selectors_id,
+                "content": "Filter on different combinations of e.g. zones, facies and regions \
+                            (The options will vary dependent on what was included \
+                            in the calculation. ",
```

Comment:
Since the `tour_steps` property has access to `self`, the tooltips could actually include the selectors included in each case. :smiley: Maybe not very important, if difficult to achieve.

Reply:
It shouldn't be too difficult to achieve, but I'm not sure how much more information it would give?
I think zone, region, facies and licenses (the available filters) are fairly well known.
For geologists atleast :)